debezium-core/src/main/java/io/debezium/jdbc/JdbcValueConverters.java

```diff
@@ -72,7 +72,10 @@
     }
 
     protected final Logger logger = LoggerFactory.getLogger(getClass());
-    private final ZoneOffset defaultOffset;
+
+    private final String zonedTimestampEpoch;
+    private final String zonedTimeEpoch;
+    protected final ZoneOffset defaultOffset;
```

Comment:
@gunnarmorling Can be back to private visibility

Reply:
Done and force-pushed.